### ISSUE TITLE
[ui5-shellbar]: unstable test

### ISSUE THREAD

**Issue by nnaydenow** [SAP/ui5-webcomponents#7340]
### Bug Description

Hi @SAP/ui5-webcomponents-topic-p, could you please investigate that test because it regularly fails and it makes the build unstable?
```
[chrome 114.0.5735.198 linux #0-11] Running: chrome (v114.0.5735.198) on linux
[chrome 114.0.5735.198 linux #0-11] Session ID: 7abb8cd547382ce50d09e43c9a7d2780
[chrome 114.0.5735.198 linux #0-11]
[chrome 114.0.5735.198 linux #0-11] » /test/specs/ShellBar.spec.js
[chrome 114.0.5735.198 linux #0-11] Component Behavior
[chrome 114.0.5735.198 linux #0-11]     Accessibility
[chrome 114.0.5735.198 linux #0-11]        ✓ tests accessibilityTexts property
[chrome 114.0.5735.198 linux #0-11]        ✓ tests acc default roles
[chrome 114.0.5735.198 linux #0-11]        ✓ tests acc custom roles
[chrome 114.0.5735.198 linux #0-11]        ✓ tests accessibilityAttributes property
[chrome 114.0.5735.198 linux #0-11]
[chrome 114.0.5735.198 linux #0-11]     ui5-shellbar menu
[chrome 114.0.5735.198 linux #0-11]        ✓ tests close on content click
[chrome 114.0.5735.198 linux #0-11]
[chrome 114.0.5735.198 linux #0-11]     ui5-shellbar-item
[chrome 114.0.5735.198 linux #0-11]        ✓ tests the stable-dom-ref attribute
[chrome 114.0.5735.198 linux #0-11]        ✓ tests count property
[chrome 114.0.5735.198 linux #0-11]        ✓ tests if shellbar item invalidates the shellbar
[chrome 114.0.5735.198 linux #0-11]        ✓ tests 'click' on custom action
[chrome 114.0.5735.198 linux #0-11]
[chrome 114.0.5735.198 linux #0-11]     Responsiveness
[chrome 114.0.5735.198 linux #0-11]        ✓ tests XXL Breakpoint 1920px
[chrome 114.0.5735.198 linux #0-11]        ✓ tests XL Breakpoint 1820px
[chrome 114.0.5735.198 linux #0-11]        ✓ tests L Breakpoint 1400px
[chrome 114.0.5735.198 linux #0-11]        ✓ tests M Breakpoint and overflow 870px
[chrome 114.0.5735.198 linux #0-11]        ✓ tests M Breakpoint and overflow 780px
[chrome 114.0.5735.198 linux #0-11]        ✓ tests M Breakpoint and overflow 710px
[chrome 114.0.5735.198 linux #0-11]        ✓ tests M Breakpoint and overflow 570px
[chrome 114.0.5735.198 linux #0-11]        ✓ tests S Breakpoint and overflow 510px
[chrome 114.0.5735.198 linux #0-11]
[chrome 114.0.5735.198 linux #0-11]     Events
[chrome 114.0.5735.198 linux #0-11]         Big screen
[chrome 114.0.5735.198 linux #0-11]            ✖ tests opening of menu
[chrome 114.0.5735.198 linux #0-11]            ✓ tests notificationsClick event
[chrome 114.0.5735.198 linux #0-11]            ✓ tests profileClick event
[chrome 114.0.5735.198 linux #0-11]            ✓ tests productSwitchClick event
[chrome 114.0.5735.198 linux #0-11]            ✓ tests logoClick event
[chrome 114.0.5735.198 linux #0-11]            ✓ tests coPilotPress event
[chrome 114.0.5735.198 linux #0-11]            ✖ tests menuItemClick event
[chrome 114.0.5735.198 linux #0-11]            ✓ tests if searchfield toggles when clicking on search icon
[chrome 114.0.5735.198 linux #0-11]            ✓ tests if searchfield toggles when altering the showSearchField property
[chrome 114.0.5735.198 linux #0-11]
[chrome 114.0.5735.198 linux #0-11]         Small screen
[chrome 114.0.5735.198 linux #0-11]            ✓ tests logoClick event
[chrome 114.0.5735.198 linux #0-11]            ✓ tests opening of menu
[chrome 114.0.5735.198 linux #0-11]            ✓ tests notificationsClick event
[chrome 114.0.5735.198 linux #0-11]            ✓ tests profileClick event
[chrome 114.0.5735.198 linux #0-11]            ✓ tests productSwitchClick event
[chrome 114.0.5735.198 linux #0-11]            ✓ tests preventDefault of productSwitchClick event
[chrome 114.0.5735.198 linux #0-11]            ✓ tests if searchfield toggles when clicking on search icon
[chrome 114.0.5735.198 linux #0-11]            ✓ tests if searchfield toggles when altering the showSearchField property
[chrome 114.0.5735.198 linux #0-11]            ✓ Shows translated label for predefined buttons, as button text when in Overflow Popover
[chrome 114.0.5735.198 linux #0-11]
[chrome 114.0.5735.198 linux #0-11] 33 passing (50.2s)
[chrome 114.0.5735.198 linux #0-11] 2 failing
[chrome 114.0.5735.198 linux #0-11]
[chrome 114.0.5735.198 linux #0-11] 1) Big screen tests opening of menu
[chrome 114.0.5735.198 linux #0-11] Menu should be shown: expected false to be truthy
[chrome 114.0.5735.198 linux #0-11] AssertionError: Menu should be shown: expected false to be truthy
[chrome 114.0.5735.198 linux #0-11]     at Context.<anonymous> (file:///home/runner/work/ui5-webcomponents/ui5-webcomponents/packages/fiori/test/specs/ShellBar.spec.js:334:12)
[chrome 114.0.5735.198 linux #0-11]     at runMicrotasks (<anonymous>)
[chrome 114.0.5735.198 linux #0-11]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[chrome 114.0.5735.198 linux #0-11]
[chrome 114.0.5735.198 linux #0-11] 2) Big screen tests menuItemClick event
[chrome 114.0.5735.198 linux #0-11] unknown error: unhandled inspector error: {"code":-32000,"message":"No node with given id found"}
  (Session info: headless chrome=114.0.5735.198)
[chrome 114.0.5735.198 linux #0-11] unknown error: unknown error: unhandled inspector error: {"code":-32000,"message":"No node with given id found"}
[chrome 114.0.5735.198 linux #0-11]   (Session info: headless chrome=114.0.5735.198)
[chrome 114.0.5735.198 linux #0-11]     at runMicrotasks (<anonymous>)
[chrome 114.0.5735.198 linux #0-11]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
------------------------------------------------------------------
```

### Expected Behaviour

Test to pass

### Steps to Reproduce

1.
2.
3.
...

### Priority

None

### UI5 Web Components Version

1.15.1

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by nnaydenow** [comment]
https://bugs.chromium.org/p/chromedriver/issues/detail?id=4440

**Comment by kgogov** [comment]
Related fix: #7677
